boto3/resources/factory.py

```diff
@@ -192,25 +163,35 @@ def _load_collections(self, attrs, model, resource_defs, service_model):
     def _load_references(self, attrs, service_name, resource_name,
```

Comment:
Is the name ``_load_references`` appropriate now that references and subresources are separate concepts and you are loading both in this function?